lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -27,6 +27,9 @@ describe 'StudentAdmin', ->
     it 'binds to the btn_rescore_entrance_exam on click event', =>
       expect(studentadmin.$btn_rescore_entrance_exam).toHandle 'click'
 
+    it 'binds to the $btn_skip_entrance_exam on click event', =>
```

Comment:
Our policy is that we don't add new CoffeeScript files, but we recognize that it doesn't make sense to have to rewrite existing tests without good reason. I have a slightly more detailed explanation of this here:

https://openedx.atlassian.net/wiki/display/TNL/How+to+add+a+new+page+to+LMS+or+Studio